### ISSUE TITLE
Unclear how to use Dropout with Sequential

### ISSUE THREAD

**Issue by peterjliu** [google/objax#162]
`objax.nn.Sequential` is a concise way to string together a bunch of layers, but it is unclear how to do this with Dropout layers, where we need to specify a `training=False/True` depending on training or inference. Just as an example
```
model = objax.nn.Sequential([
    objax.nn.Linear(1, 20),
    objax.nn.Dropout(0.5),
    objax.nn.Linear(20, 1)])
```

It appears you can do this:
```
model(x, training=False)   # pass in dropout boolean
```

But it wasn't clear from the documentation. This may be a good example to add. Also please let me know whether this works as intended.

**Comment by david-berthelot** [comment]
Yes it does work the way you described, I've made a blog post about Sequential, linking it in case you find additional information in it: https://david-berthelot.github.io/2020-10-07-sequential-tinkering-part-1/

Note, there's already an example in the documentation https://objax.readthedocs.io/en/latest/objax/nn.html#objax.nn.Sequential

We keep adding to the docs all the time, so it's always worth checking if it changed since your last visit. Let me know if the docs are not clear enough or if you have suggestions for changes.

Thanks!

**Comment by peterjliu** [comment]
Ah, I didn't see the code example. I would recommend adding some text somewhere, perhaps in the API description. I did see your blog-post but it didn't have such an example exactly. Having a dropout example, which is a very common use-case for pretty much all models might make things clearer.
